modules/hmftools/1.0/hmftools.smk

```diff
@@ -39,40 +39,49 @@ if version.parse(current_version) < version.parse(min_oncopipe_version):
 # `CFG` is a shortcut to `config["lcr-modules"]["hmftools"]`
 CFG = op.setup_module(
     name = "hmftools",
-    version = "1.0",
-    subdirectories = ["inputs", "prepare_strelka", "amber", "cobalt", "purple", "linx", "outputs"],
+    version = "1.1",
+    subdirectories = ["inputs", "prepare_slms3", "amber", "cobalt", "purple", "linx", "outputs"],
 )
 
 # Define rules to be run locally when using a compute cluster
 localrules:
     _hmftools_input_bam,
-    _hmftools_input_strelka,
-    _hmftools_strelka_sample_names, 
+    _hmftools_input_slms3,
+    _hmftools_slms3_sample_names, 
     _hmftools_input_gridss, 
     _hmftools_input_references,
     _hmftools_get_cobalt_gc,
+    _hmftools_get_cobalt_bed, 
     _hmftools_get_amber_snps, 
     _hmftools_get_purple_drivers, 
     _hmftools_get_linx_db,
-    _hmftools_linx_prepare_ensembl,
+    _hmftools_get_ensembl_cache, 
     _hmftools_purple_output,
     _hmftools_purple_plots,  
     _hmftools_all
 
 
-HMFTOOLS_VERSION_MAP = {
-    "grch37": "hg19",
-    "hs37d5": "hg19",
-    "hg38": "hg38"
+VERSION_MAP_HMFTOOLS = {
+    "grch37": "37",
+    "hs37d5": "37",
+    "hg38": "38"
 }
 
-possible_genome_builds = HMFTOOLS_VERSION_MAP.keys()
+possible_genome_builds = VERSION_MAP_HMFTOOLS.keys()
 for genome_build in CFG["runs"]["tumour_genome_build"]:
     assert genome_build in possible_genome_builds, (
         "Samples table includes genome builds not yet compatible with this module. "
         "This module is currently only compatible with {possible_genome_builds}. "
     )
 
+wildcard_constraints: 
```

Comment:
this looks like global constraints - just checking if this is intended, I thought we're trying to avoid it and specify these inside the rules. I think the rules below do have constraints specified, too